bin/newver

```diff
@@ -7,11 +7,19 @@ if [ $# -eq 0 ]; then
 fi
 NEWVER="$1"
 
+previous_channel() {
+  ruby -e "version = \"${NEWVER}\".match(/(\d+).(\d+).(\d+)/); \
+    minor = version[2].to_i - 1; \
+    puts \"channel/rubocop-#{version[1]}-#{minor}\""
```

Comment:
This is very clever. IMO it wouldn't be crazy to rewrite this whole script in ruby with a `#!/usr/bin/env ruby` shebang if that will make this all easier

Happy to stick with this, though. It might make your life a little easier to do something like:

```shell
cat <<EOM | ruby
  ruby goes here without need of escaping quotes
EOM
```